app/controllers/videos_controller.rb

```diff
@@ -1,10 +1,14 @@
 class VideosController < ApplicationController
-  after_action :verify_authorized
+  after_action :verify_authorized, except: %i[index]
 
```

Comment:
Check out the `pages_controller` for the simplest example of applying edge caching to the page. (`set_cache_control_headers` and `set_surrogate_key_header`)

Then the page will need to be busted at appropriate times, like any other page.